tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1046,6 +1046,31 @@ def test_submissions(self):
             for i in range(101):
                 assert isinstance(next(generator), Submission)
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_pause_comments(self, _):
+        with self.recorder.use_cassette('TestSubredditStreams.'
+                                        'comments_pause'):
+            stream = self.reddit.subreddit('kakapo').stream
+            with_pause = stream.comments(pause_after=0)
+            for i in range(10):
```

Comment:
This might never actually hit a pause because the first 100 items should all be comments. I think, however, it's fine to delete this test and rely just on the other one since it includes a pause.